.rpmlintrc

```diff
@@ -0,0 +1 @@
+addFilter('macro-in-comment')
```

Comment:
IMHO we should clean these up but I'm ok with merging it if it gives us other lint checks in the short term. Don't we currently run rpmlint now?

Reply:
We do, this hides this warning from the output. I'd prefer a model of hiding warnings via rpmlint config and then removing them as we/if we want to fix them.

Reply:
Can we make warnings fatal? Currently warnings don't cause a build failure so they're ignored, even when shown.

Reply:
Does not look like we can. But we can at least hide this output since I think there is an actual error that can surface related to macros being expanded in comments.

Reply:
Can we check if stdout/stderr is empty perhaps?

Reply:
Probably can check for and error out in Obal if a warning exists easily. This however, I think we want to ignore all together since we use macros in comments for documentation and we'd have to remove that use everywhere.